### ISSUE TITLE
 Permission error in Claude3

### ISSUE THREAD

**Issue by linkedlist771** [Amm1rr/WebAI-to-API#29]
Following the readme steps , got permission error.

![image](https://github.com/Amm1rr/WebAI-to-API/assets/72634327/0486aa6e-6f5a-4bc1-b69b-c557a266d516)

**Comment by linkedlist771** [comment]
I login to the browser through my local machine and run this app in a remote VPS.  This may be the reason for this, for I want to use the VPS to provide a long time service. I have tried to login to the `Chrome` and `Edge` in my local machine with different accounts, which is fine.  And my VPS is a headless servicer, I have no idea about to use it to login in.

**Comment by Amm1rr** [comment]
You have two options to address the issue:
- 1. The problem lies in logging into your VPS via a browser instead of your local machine.
- 2. Alternatively, you can configure your _local machine session_ in the **Config.conf** file on your _VPS_.

**Comment by linkedlist771** [comment]
Yes indeed, I choose the second options to use this.  Here is what I do to trigger this problem:
1. I log in to my Claude 3 account in Chrome on my local machine. 
2. I use `F12` to find the cookie in the browser, I paste it into the `Config.conf`.
3. I start this project on my VPS use the `Config.conf` in step 2.
4. I use the docs route provided by fastapi in the service in step3 to make a request. 

Then I met this issue, got `permission denied`.  Maybe the cookie is related with login `IP`, later I will try to use this project on my local machine to make sure of it.

**Comment by Amm1rr** [comment]
Update your clone, then run the server and check your sessions on [http://localhost:8000/WebAI](http://localhost:8000/WebAI)
Can you see sessions for Claude and Gemini?

**Comment by linkedlist771** [comment]
After all thoes methods I tried, finally I succeed. It  turns out the the `login in device` seems to be the key issue in my case. The cookie is bound with the `IP` address(or maybe the device) when you first login into it. So it can not be used elsewhere except the login device. By the way, I want to know that, what is the `expiration date` of the cookie, if it is possible to have a auto-login/refresh module here.  

![image](https://github.com/Amm1rr/WebAI-to-API/assets/72634327/42613400-e4ec-4784-ad42-aa392d818904)
